steward/src/commands/eth_to_cosmos.rs

```diff
@@ -14,24 +14,49 @@ const TIMEOUT: Duration = Duration::from_secs(60);
 
 /// Send Ethereum to Cosmos
 #[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Send Eth token to Cosmos chain.\n This command sends Eth token to the Cosmos chain via the Gravity bridge. \n It takes the tx amount, Eth keyname, contract address, Cosmos token destination, number of times \n transaction should be made and if the transaction should be made immediately or wait for the next \n batch."
+)]
 pub struct EthToCosmosCmd {
-    pub args: Vec<String>,
+    /// H160 Erc20 contract address.
+    #[clap(short = 'E', long)]
+    erc20_address: String,
+
+    /// Tx amount.
+    #[clap(short, long)]
+    init_amount: String,
+
+    /// Eth keyname, takes a String.
+    #[clap(short, long)]
+    ethereum_key: String,
+
+    /// H160 Cosmos address
```

Comment:
Aside from the general comment around not needing to specify `H160`, Cosmos addresses will not be in that format anyway.